src/mono/sample/wasm/Directory.Build.targets

```diff
@@ -39,11 +39,12 @@
       <_Dotnet>$(RepoRoot)dotnet$(_ScriptExt)</_Dotnet>
       <_AOTFlag Condition="'$(RunAOTCompilation)' != ''">/p:RunAOTCompilation=$(RunAOTCompilation)</_AOTFlag>
```

Comment:
It would be cleaner to combine all flags into a single `_PublishArgs` property

Reply:
It would, next time we touch it we can set the unset ones to `false` and pass them all.